setup.py

```diff
@@ -77,8 +77,8 @@ def package_files(prefix: str, directory: str, extension: str):
     'tqdm>=4.62.3,<5',
     'torchmetrics>=0.10.0,<1.1',
     'torch_optimizer>=0.3.0,<0.4',
-    'torchvision>=0.13.1,<0.16',
-    'torch>=1.13.1,<2.1',
+    'torchvision>=0.13.1,<=0.16',
+    'torch>=1.13.1,<=2.1',
```

Comment:
```suggestion
    'torchvision>=0.13.1,<0.17',
    'torch>=1.13.1,<2.1.1',
```
Minor nit: Can we use < pin to be consistent?

Reply:
thanks fixed